### ISSUE TITLE
Announcement Post - Abigail Mesrenyame Dogbe Wins Inaugural Outstanding PyLady Award

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/blackpythondevs.github.io#330]
Create Announcement Post with the following Format

- the announcement
- what is Pyladies
- what is the outstanding pylady award
- when was the award given (On stage at PyCon 2024 in Pittsburgh,PA. 
- Include quote from Dogbe (Requested)
- Include quote from PyLadies
- Include quote from BPD Executor (to be provided)

**Comment by kjaymiller** [comment]
The post is ready and we are waiting for an update from PyLadies to release this announcement.
